tests/ProcessManager/ChromeManagerTest.php

```diff
@@ -68,4 +68,22 @@ public function testMultipleInstances()
         $driver1->quit();
         $driver2->quit();
     }
+
+    public function testExtensionsAndExperimentalOptions()
+    {
+        $manager = new ChromeManager();
+        $manager->setExtensions([
+            __DIR__.'/../fixtures/chrome/extension1.crx',
+            __DIR__.'/../fixtures/chrome/extension2.crx',
+        ]);
+        $manager->setExperimentalOptions([
+            'excludeSwitches' => ['enable-automation'],
+            'mobileEmulation' => [
+                'deviceName' => 'Pixel 2',
+            ],
+        ]);
+        $client = $manager->start();
+        $this->assertNotEmpty($client->getCurrentURL());
```

Comment:
Is it possible to add further assertions to verify that the expected extensions have been set and that the expected experimental options are in effect?

Reply:
Currently Chrome headless mode does not support extension, so it's not possible to verify that the extension is enabled. We can use `xvfb-run` like [this](https://stackoverflow.com/a/46475980/2622199), but I think it will make thing more complicated.